docs/examples/node_postprocessor/JinaRerank.ipynb

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
<p>could you print out the reranked nodes from response </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/11291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
how is this printed?